packages/utils/src/getActiveLoginIDFromLocalStorage.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Get active loginid from localStorage
+ * @returns {string} - Active loginid
+ */
+const getActiveLoginIDFromLocalStorage = (): string => {
+    const active_loginid = localStorage.getItem('active_loginid') || '';
```

Comment:
```suggestion
    const active_loginid = localStorage.getItem('active_loginid');
```
@sergei-deriv By giving it a default empty string in the code TS won't ask us to do null check which might lead to buggy code later.